### DISCUSSION TITLE
Nginx reverse proxy not getting js and css files

### DISCUSSION THREAD

**Discussion by Stevenjohnlally** [firefly-iii/firefly-iii#4468] [Firefly III]
I am running Firefly III version 5.4.6 with docker.

**Description**
I have a domain with duckdns so that I can expose my service to the public. I'll use _example.duckdns.org_ as the placeholder here because I don't want to let just anyone use it.

I have the firefly container's port 8080 attached to port 49152 of the host machine.

I have set up NginX as a reverse proxy on the host machine so that any requests that look like this, _http://example.duckdns.org/firefly_, will be forwarded to the docker container.

Here is my NginX config server block.
```
server {
        listen 80;

        add_header Referrer-Policy "no-referrer";

        location /firefly/ {
                proxy_pass http://localhost:49152/;
                proxy_set_header Host $http_host;
                proxy_set_header X-Real-IP $remote_addr;
                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                proxy_set_header X-Forwarded-Proto $scheme;
                proxy_buffering off;
        }
}
```
I can load the login page fine by going to _http://example.duckdns.org/firefly/login_

I inspected the web app and see that the base href is set to _http://example.duckdns.org/_

Therefore the assets are requested as _http://example.duckdns.org/v1/lib/bs/css/bootstrap.min.css?v=5.4.6_
instead of _http://example.duckdns.org/firefly/v1/lib/bs/css/bootstrap.min.css?v=5.4.6_ which would work fine.

How can I set it up so that the base href is set to _http://example.duckdns.org/firefly/_?

**Comment by JC5** [comment]
Not with the default Docker image. It only supports reverse proxies from the root directory.
